Scripts/Haptics/HapticPulseFromAudioClip.cs

```diff
@@ -0,0 +1,121 @@
+﻿namespace VRTK.Core.Haptics
+{
+    using UnityEngine;
+    using System.Collections;
+
+    /// <summary>
+    /// Creates a haptic pattern based on the waveform of an <see cref="AudioClip"/> and utilizes a <see cref="HapticPulse"/> to create the effect.
+    /// </summary>
+    public class HapticPulseFromAudioClip : HapticProcess
+    {
+        /// <summary>
+        /// The pulse process to utilize.
+        /// </summary>
+        [Tooltip("The pulse process to utilize.")]
+        public HapticPulse hapticPulse;
+        /// <summary>
+        /// The waveform to represent the haptic pattern.
+        /// </summary>
+        [Tooltip("The waveform to represent the haptic pattern."), SerializeField]
+        protected AudioClip audioClip;
+        /// <summary>
+        /// Multiplies the current audio peak to affect the wave intensity.
+        /// </summary>
+        [Tooltip("Multiplies the current audio peak to affect the wave intensity."), SerializeField]
+        protected float intensityMultiplier = 1f;
+
+        /// <summary>
+        /// The waveform to represent the haptic pattern.
+        /// </summary>
+        public AudioClip AudioClip
```

Comment:
Either just offer the field or fix the formatting of the get+set here to be a single line.

Reply:
👍